tests/causal_estimators/test_causalml_estimator.py

```diff
@@ -0,0 +1,216 @@
+import pytest
+import subprocess
+import sys
+
+from dowhy import CausalModel
+from dowhy.datasets import linear_dataset
+
+# To skip the tests if we cannot install causalml
+installed_failed = False
+# Hack to install causalml if not already present
+try:
+    __import__("causalml")
+except ImportError:
+    try:
+        subprocess.check_call([sys.executable, "-m", "pip", "install", "causalml"])
+        __import__("causalml")
+        # We import this later as we obtained this from causalml

```

Comment:
what is "this"? Assuming that you mean XGBregressor?

Reply:
yes